### PR TITLE
Download GNU parallel directly from git

### PR DESCRIPTION
**Pull request by dra27**
cf. https://github.com/ocaml/ocaml/pull/10760#issuecomment-1029356670 (with thanks to GNU parallel's author, @ole-tange!)

### CODE REVIEW

File: tools/ci/appveyor/appveyor_build.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ function set_configuration {
 #    run "Content of $FILE" cat Makefile.config
 }
 
-PARALLEL_URL='https://ftpmirror.gnu.org/parallel/parallel-latest.tar.bz2'
+PARALLEL_URL='https://git.savannah.gnu.org/cgit/parallel.git/tree/src/parallel'
```
Comment (kit-ty-kate) [inline tools/ci/appveyor/appveyor_build.sh]:
```suggestion
PARALLEL_URL='https://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel'
```

Comment (kit-ty-kate) [reply tools/ci/appveyor/appveyor_build.sh]: this url returned html for me. With this change it should return plain text

Comment (dra27) [reply tools/ci/appveyor/appveyor_build.sh]: Oops - hopefully corrected version pushed, thanks!

**Review by gasche — APPROVED** [review]
This is about Appveyor, from David, and Kate already looked at it. I don't want to know what's inside, but we're probably better off with it merged than not merged. Approved.

**Pull request merged**